library/src/test/java/com/chuckerteam/chucker/internal/support/JsonConverterTest.kt

```diff
@@ -32,6 +33,27 @@ class JsonConverterTest {
         assertThat(dateTestClass).isEqualTo(DateTestClass(Date(0)))
     }
 
+    @Test
+    fun testMoshiConfiguration_willSerialize_null_DateTime() {
+        val dateTestJson = JsonConverter.instance.adapter(DateTestClass::class.java)
+            .toJson(DateTestClass(null))
+        assertThat(dateTestJson).isEqualTo("{}")
+    }
+
+    @Test
+    fun testMoshiConfiguration_willDeserialize_null_DateTime() {
+        val dateTestClass = JsonConverter.instance.adapter(DateTestClass::class.java)
+            .fromJson("""{"date":null}""")
+        assertThat(dateTestClass).isEqualTo(DateTestClass(null))
+    }
+
+    @Test(expected = JsonDataException::class)
+    fun testMoshiConfiguration_willDeserialize_Invalid_DateTime() {
+        val dateTestClass = JsonConverter.instance.adapter(DateTestClass::class.java)
+            .fromJson("""{"date":"abcd"}""")
```

Comment:
Why you created this test? 
It is an impossible case to have such string to be converted with DateAdapter except explicitly doing it here. 
What are you testing here?

Reply:
Removed code related to `Date` parsing as it is not used anywhere in the codebase.